Source/ProgressionSystemRuntime/ProgressionSystemRuntime.Build.cs

```diff
@@ -23,7 +23,7 @@ public ProgressionSystemRuntime(ReadOnlyTargetRules Target) : base(Target)
 				// Bomber modules 
 				"MyUtils" 
 				, "PoolManager" // Star and Widget Actors
-				, "MetaCheatManager" // PSCheatExtension
+				, "MetaCheatManager",  // PSCheatExtension
```

Comment:
Please revert this unnecessary change

Reply:
done as part of https://github.com/h4rdmol/ProgressionSystem/pull/12/commits/0dc42fbdf6ceb67efd71cbafead115ade2be8ad1 and https://github.com/h4rdmol/ProgressionSystem/pull/12/commits/205b74a50da8effcf5f18761ff1eb503c2a7abd7